content/docs/40-client-js/20-event/index.md

```diff
@@ -59,17 +59,16 @@ client.event.emit('notifications', 'Maria just came online')
 |Argument|Type|Optional|Description|
 |---|---|---|---|
 |pattern|String (regex)|false|The pattern to match events which subscription status you want to be informed of|
-|callback|Function|false|A function that will be called whenever an event has been initially subscribed to or is no longer subscribed. Arguments are (String) match, (Boolean) isSubscribed, and response (Object)|
+|callback|Function|false|A function that will be called whenever an event has been initially subscribed to or is no longer subscribed. Arguments are (String) match, and response (Object)|
```

Comment:
>  to or is no longer subscribed

Function does not get called, the onStop function does.

Reply:
Committed a fixed phrasing.